app/assets/stylesheets/sass/pages/employees.scss

```diff
@@ -0,0 +1,23 @@
+@charset 'utf-8';
+
+.bg-table {
+  background-color: #435d7d;
```

Comment:
I you are using a pattern like [this](https://sass-guidelin.es/#the-7-1-pattern), I recommend you to create the 
`abstracts /_variables.scss` or `abstracts /_colors.scss`  and create your color_variables there, for example
```
$primary-color: #435D7D;
```
and now in: https://github.com/BrightCoders-Proyectos/control-almacen/pull/21/files#diff-8820054d1c0d00787544b1e096c47164R4
you can do something like this:
```
background-color: $primary-color;
```
and don't forget to import this file in `app/assets/stylesheets/sass/main.scss` before the file that is going to use that variable

the same for this:
https://github.com/BrightCoders-Proyectos/control-almacen/pull/21/files#diff-8820054d1c0d00787544b1e096c47164R16

Reply:
Colors are not defined yet, I agree is better use variables I'll change it for $bg-gray-table: #435D7D; and then when colors get defined I'll change the name